tests/data/samplesheets/samplesheet-campy-staph.csv

```diff
@@ -0,0 +1,2 @@
+sample,fastq_1,fastq_2,long_reads,assembly
+CSE,https://github.com/phac-nml/mikrokondo/raw/integration-testing/tests/data/reads/campy-staph1.fq.gz,https://github.com/phac-nml/mikrokondo/raw/integration-testing/tests/data/reads/campy-staph2.fq.gz,,
```

Comment:
Maybe switch from using `integration-testing` in the URL to the commit ID (I recall you using the commit ID before)? That way, these would still work when that branch is deleted, and later on we can update to point to the data on `dev`?

Reply:
I switched it over to dev, but let me know if you want it to the commit ID instead: 17902044f3f94bf304e59aa522f6b03aaf48fb3c

Reply:
If `dev` works that sounds good to me.